inspirehep/accounts/api.py

```diff
@@ -26,3 +26,14 @@ def is_superuser_or_cataloger_logged_in():
         user_roles = {role.name for role in current_user.roles}
         return user_roles & {"superuser", "cataloger"}
     return False
+
+
+def login_required_with_role_superadmin_or_cataloger(func):
```

Comment:
maybe it's better to have sth `login_required_with_roles` or have `roles` param for the `login_required` that defaults to `all`/`user`

Reply:
done